### ISSUE TITLE
Resolve the error

### ISSUE THREAD

**Issue by Amit366** [Tech-N-Science/FunwithScience#335]
**Describe the bug**
On clicking the inspect of the website you see two errors, resolve that. 

![Screenshot (28)](https://user-images.githubusercontent.com/60662775/151663921-ba268ca5-c0d1-48de-866b-4a196cda7468.png)

**Comment by SabariGanesh-K** [comment]
Assign this issue, will try it out👍👏

**Comment by SabariGanesh-K** [comment]
Assign it to me . I have guessed some probable reasons why its crashing . Will analyse and will fix it

**Comment by SabariGanesh-K** [comment]
Basically this issue is due to issue from server side and not client side.
Actually in the server of adsense , CORS needs a header (Access control allow origin) to allow 
this website and also localhost.
![image](https://user-images.githubusercontent.com/64348740/152326921-b11b8ec5-7f5a-4c14-a1a1-5780bfe84d20.png)

Visit this link to know about it:-
But the above process cannot be done by us as only client side is in our hands. So if the error is occurring , its due to the fact
=> Publisher ID might be incorrect
=> ads are not being added to website.
=> Nest hub devices have restriction over auto ads
To verify what I am saying , you may install this extension:-
https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf/related
And turn the extension on
like this:-
![image](https://user-images.githubusercontent.com/64348740/152327340-de01cc78-aab9-443b-91c9-f6d15bf3ba9b.png)

And after its turned on, you may reload the website. And you can see
![image](https://user-images.githubusercontent.com/64348740/152327670-362f1e8a-203b-4a03-90a4-15664fb74867.png)
the first error (Access to XMLhtml ...) is gone.
But this is temporary as this extension would default allow  things needed(access control allow origin to all websites) and when uninstalled or disabled the error will be back.

So basically this as far my knowledge and above resource goes, it cannot be controlled through client side.

**Comment by Amit366** [comment]
ok no problem. Thanks for pointing out a solution atleast. @SabariGanesh-K
